src/main/java/main/responses/EmptyPathResponse.java

```diff
@@ -3,13 +3,17 @@
 import main.Request;
 import main.Response;
 
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 import static main.Status.METHOD_NOT_ALLOWED;
 import static main.Status.OK;
 
 public class EmptyPathResponse extends DefaultResponse {
     private final List<String> content;
+    String publicDirectory = "/Users/priyapatil/cob_spec/public";
```

Comment:
Isn't this going to work on your machine only? You need an abstraction around the file system.